targets/esp32/jshardware.c

```diff
@@ -354,8 +341,9 @@ void jshSetOutputValue(JshPinFunction func, int value) {
   }
   else{
     pin = ((func >> JSH_SHIFT_INFO) << 4) + ((func >> JSH_SHIFT_TYPE) & 15);
-    value >> (16 - PWMTimerBit);
-    setPWM(pin,value);
+    // Was warning: statement with no effect [-Wunused-value] - changed to value - needs testing!
```

Comment:
@jumjum123 

I have commented this out as it has no effect and was a compiler warning - the results was thrown away so I'm not sure this code is working as intended?

Can you recheck  it working as intended?